### DISCUSSION TITLE
Is there way to add the white border line around floating window

### DISCUSSION THREAD

**Discussion by SumitAchaju** [LazyVim/LazyVim#2058] [Q&A]
Is there way to add the white border line around floating window which helps to distinguish between background and foreground in transparent neovim mode

**Comment by dpetka2001** [comment]
Add 
```lua
return {
  {
    "folke/noice.nvim",
    opts = {
      presets = {
        lsp_doc_border = true, -- add a border to hover docs and signature help
      },
    },
  },
}
```
to a file in `~/.config/nvim/lua/plugins/noice.lua` and see if that's what you want.

**Reply by wigging** [reply]
This works for me and it should be the accepted answer.

**Comment by RicardoRyn** [comment]
`vim.o.winborder = "single"`
It's a bit hacky, not perfect, but simple, and I've been using it for a long time.
